src/components/ActionMenu/components/Actions/Actions.tsx

```diff
@@ -183,7 +195,11 @@ export function Actions({actions = [], groups = []}: Props) {
       return groups.length === 0 && group === defaultRollupGroup
         ? group
         : group === lastMenuGroup ||
-            (group !== defaultRollupGroup && !rolledUpActions.includes(group));
+            !rolledUpActions.some(
+              (rolledUpGroup) =>
+                isMenuGroup(rolledUpGroup) &&
+                rolledUpGroup.title === group.title,
```

Comment:
Made this more strict because sometimes `includes` was reading as a false positive.